specs/adr/0006-misc.md

```diff
@@ -0,0 +1,18 @@
+# Overall Page
+* Status: proposed <!-- optional -->
+* Deciders: whole team <!-- optional -->
+* Date: 2021-05-20 <!-- optional -->
+## Context and Problem Statement
+Which browsers are we focused on supporting with our web application?
+## Considered Options
+* Chromium only
+* Native browsers (Safari, Edge, etc.)
+* Chromium and Safari
+## Decision Outcome
+Chosen option: "Chromium only", because most college students use Chromium browsers and they are our target audience.  
```

Comment:
Would it be alright for us to link a resource for this to support our decision?

Reply:
https://gs.statcounter.com/